src/decode.ts

```diff
@@ -187,7 +189,7 @@ export function decryptScalar(
     case "float":
       return parseFloat(cleartext);
     case "bool":
-      return cleartext === "true";
+      return cleartext === "True";
```

Comment:
Can we change this to be case insensitive, matching `strconv.ParseBool(...)`

Reply:
return cleartext.toLowerCase() === "true";   we can do like this for case insensitive

Reply:
Work for me.

Reply:
I updated the PR